ios-base/DroidKaigi 2020/Sidebar/Views/SidebarViewController.swift

```diff
@@ -43,13 +43,33 @@ final class SidebarViewController: UITableViewController {
 
         switch switchType {
         case .timeline:
-            rootViewController?.navigationDrawerController?.toggleLeftView()
+            if let _ = UIApplication.topViewController() as? FilterViewController {
```

Comment:
Is `rootViewController` property useless?

Reply:
It's miss.
Fix reverted